test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -43,6 +44,7 @@ const (
 	configFile        = "config/performance-config.yaml"
 	createNodesOpcode = "createNodes"
 	createPodsOpcode  = "createPods"
+	churnOpcode       = "churn"
```

Comment:
let's name this in a way that test configurations are more readable. Perhaps `continuosRecreates` or `noiseRecreates`.

Reply:
We used `churn` a lot and it's the most readable wording to me :) But I'm not a naming expert, @ahg-g @adtac any naming thoughts?

Reply:
I prefer churn too. Maybe "backgroundChurn" to emphasise that this is non-blocking? would also somewhat cover @alculquicondor's suggestion

Reply:
```
  - opcode: churn
    serviceTemplatePath: config/churn/service-default.yaml
    intervalMilliseconds: 1000
```

this doesn't make sense to me. It almost feel like networking churn. Maybe `recreatesChurn`

Reply:
The pure-service churn is an example of evaluating how Service events may or may not churn the cluster. I removed it to postpone the debate of how practical it could be - I'd like to get to a consensus of how a churn op looks like first, and in follow-up PRs we can add/discuss test cases on demand, also how wild the test case can churn the cluster.

The current logic is to create & delete _customizable_ API objects interleavingly. Other options are:

- create & delete **N** API objects interleavingly
- keep creating API objects (without deleting them)

@alculquicondor @adtac WDYT?

Reply:
can we have something like this:

```
  - opcode: apiChurn
    mode: recreate
    serviceTemplatePath: config/churn/service-default.yaml
    intervalMilliseconds: 1000
```

Note that I'm not referring to the specific test case, just how the test description and configuration looks like.

Reply:
SG. So we have different churn modes/patterns to choose from.

- recreate (or continuous-create)
- ephemeral-create (create & delete a single object)
- round-robin-create (create N objects, and then delete N objects, N can be configurable)
- and probably incorporate update events

BTW: when you say `recreate`, do you mean continuously creating API objects?